documentation/sphinx/source/release-notes.rst

```diff
@@ -93,7 +93,8 @@ Other Changes
 * Improved the slow task profiler to also report backtraces for periods when the run loop is saturated. `(PR #2608) <https://github.com/apple/foundationdb/pull/2608>`_
 * Double the number of shard locations that the client will cache locally. `(PR #2198) <https://github.com/apple/foundationdb/pull/2198>`_
 * Replaced the ``-add_prefix`` and ``-remove_prefix`` options with ``--add_prefix`` and ``--remove_prefix`` in ``fdbrestore`` `(PR 3206) <https://github.com/apple/foundationdb/pull/3206>`_
-* Data distribution metrics can now be read using the special key-space ``\xff\xff/metrics/data_distribution_stats``. `(PR #2547) <https://github.com/apple/foundationdb/pull/2547>`_
+* Data distribution metrics can now be read using the special keyspace ``\xff\xff/metrics/data_distribution_stats``. `(PR #2547) <https://github.com/apple/foundationdb/pull/2547>`_
```

Comment:
Right now, this is only `Shard byte size can be read using the special keyspace`. The other data distribution metrics will be added in future releases. 

I think it's better to be more specific so that readers of the doc won't get confused that all DD metrics have been exposed.

Reply:
Hmm, I'm not the author of this line (I just made a minor adjustment) so I'm not sure what the best way to represent this is. It feels like ideally, we'd have some documentation of this section in our official docs, and then we'd link to that from here such that the contents of the section could be explored. Does anything like that exist?

Reply:
I reviewed that PR. :)
I don't think we have the official doc yet, which I agree is a good idea.